### PR TITLE
Add README.CN.md

### PR DESCRIPTION
**Pull request by colachg**

### CODE REVIEW

File: README.CN.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- 如下图连接 GPIO 串口的 14/15 号引脚
+- [第六章](06_drivers_gpio_uart) 是这个设备第一次需要使用的地方。找到如何准备 SD 卡来引导你自制的内核的说明。
+- [第七章](07_uart_chainloader)开始，在树莓派上启动内核变得非常舒适。在这章，会开发出一个叫`chainloader`的文件。
+  这将是您暂时需要在 SD 卡上手动复制的最后一个文件。这将使您能够在通过 UART 按需引导期间加载教程内核。
+
+![UART wiring diagram](doc/wiring.png)
+
+## 🙌 致谢
+
+这个教程最初是有[该项目](https://github.com/bztsrc/raspi3-tutoria)衍生出来的，感谢它给我开了一个头。
```
Comment (Disasm) [inline README.CN.md]: `https://github.com/bztsrc/raspi3-tutorial` (last letter `l` was missing)

Comment (colachg) [reply README.CN.md]: Yes, thank you. I will fix it.

**Comment by andre-richter** [comment]
:+1: 

Pending review from @readlnh.

File: README.CN.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+[armv8-a 架构]: https://developer.arm.com/products/architecture/cpu-architecture/a-profile/docs
+[功能健全的]: https://en.wikipedia.org/wiki/Monolithic_kernel
+
+## 📑 教程结构
+
+- 每篇教程都包含一个独立可引导的二进制内核文件。
+- 每篇新的教程都在之前的基础上扩展。
+- 每篇教程的指南里面都有一个简短的章节来总结新增的代码和功能，也会展示源代码的区别，方便检查和同步。
+- 当我认为源代码的变更不能够表现我的想法时，我会在教程里用非常详细和具体的文字介绍。
```
Comment (readlnh) [inline README.CN.md]: Is this translation omitted too much?

File: README.CN.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+[visual studio code]: https://code.visualstudio.com
+[rust language server]: https://github.com/rust-lang/rls
+
+## 🚀 易用性
+
+这个系列的教程会着重关注用户体验的友好性。因此，我尽量消除嵌入式开发中的最大痛点：工具链的问题。
+
+想要尝试代码的开发者们将不会因为复杂工具链的安装、编译而烦恼。这是尽量依靠 rust 的标准工具库和通过
+docker 容器提供的其他工具库实现的。开发者只需要安装好 docker ，容器在需要时会被自己运行。
```
Comment (readlnh) [inline README.CN.md]: 想要尝试代码的开发者们将不会因为复杂工具链的安装、编译而烦恼。这是通过尽可能的使用 rust 的标准工具链并由相应的 docker 容器来提供其他所有的工具链实现的。该容器会在需要的时候自动下载，开发者只需要安装好对应发行版的 docker 即可。

File: README.CN.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+- 如下图连接 GPIO 串口的 14/15 号引脚
+- [第六章](06_drivers_gpio_uart) 是这个设备第一次需要使用的地方。找到如何准备 SD 卡来引导你自制的内核的说明。
+- [第七章](07_uart_chainloader)开始，在树莓派上启动内核变得非常舒适。在这章，会开发出一个叫`chainloader`的文件。
+  这将是您暂时需要在 SD 卡上手动复制的最后一个文件。这将使您能够在通过 UART 按需引导期间加载教程内核。
+
+![UART wiring diagram](doc/wiring.png)
+
+## 🙌 致谢
+
+这个教程最初是有[该项目](https://github.com/bztsrc/raspi3-tutorial)衍生出来的，感谢它给我开了一个头。
```
[the remark this answers is not in the document]
Comment (colachg) [reply README.CN.md]: Ok, suggestions accepted. Thank you.

**Comment by andre-richter** [comment]
Awesome, thanks!

**Pull request merged**